src/PaymentCard.php

```diff
@@ -27,21 +27,11 @@ class PaymentCard extends Model
         'last_four',
         'expiry_month',
         'expiry_year',
-        'cvv',
-        'type',
-        'is_primary',
```

Comment:
Where did "is_primary" go? Or will it be handled by the code where you integrate the library?

Reply:
Yes, developer should be extended this model and define his own fields like `is_primary`, `type`, etc. I've already extended it in model of our project.